packages/mobile/src/navigator/TabNavigator.tsx

```diff
@@ -42,7 +42,7 @@ const mapStateToProps = (state: RootState): StateProps => ({
 class WalletIconWithCircle extends React.Component<Props> {
   render() {
     return (
-      <View>
+      <View style={{ marginLeft: 3 }}>
```

Comment:
🚨🚨🚨

Reply:
Thanks - forgot to push commit fixing this